app/electron/main.js

```diff
@@ -176,14 +193,9 @@ app.on('quit', function () {
   }
 });
 
-app.on('activate', function () {
-  if (mainWindow === null) {
-    createWindow();
-  }
-});
-
-app.once('window-all-closed', app.quit);
-
-app.once('before-quit', () => {
-  mainWindow.removeAllListeners('close');
-});
+const isHeadlessMode = process.argv[1] === 'headless';
```

Comment:
Please use `yargs` for this. Let's give it a nice description and have the base so we can add other options later (like the `port`).

Reply:
We also want to use it as an option (`--headless`), not a simple argument.